i18n/language/pluralspec.go

```diff
@@ -339,6 +339,17 @@ var pluralSpecs = map[string]*PluralSpec{
 		},
 	},
 
+	// Turkish
+	"tr": &PluralSpec{
+		Plurals: newPluralSet(Zero, Other),
```

Comment:
This logic doesn't seem consistent with:
http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html

Can you double check this?